lib/android/app/src/main/java/com/reactnativenavigation/views/TopTabsLayout.java

```diff
@@ -4,35 +4,43 @@
 import android.content.Context;
 import android.support.annotation.RestrictTo;
 import android.support.v4.view.ViewPager;
+import android.view.View;
+import android.view.ViewGroup;
 import android.widget.LinearLayout;
+import android.widget.RelativeLayout;
 
 import com.reactnativenavigation.parse.NavigationOptions;
 import com.reactnativenavigation.presentation.OptionsPresenter;
+import com.reactnativenavigation.utils.Task;
 import com.reactnativenavigation.viewcontrollers.toptabs.TopTabController;
 import com.reactnativenavigation.viewcontrollers.toptabs.TopTabsAdapter;
 import com.reactnativenavigation.viewcontrollers.toptabs.TopTabsViewPager;
 
 import java.util.List;
 
 @SuppressLint("ViewConstructor")
-public class TopTabsLayout extends LinearLayout implements Container {
+public class TopTabsLayout extends RelativeLayout implements Container {
 
     private TopBar topBar;
+    private List<TopTabController> tabs;
     private TopTabsViewPager viewPager;
     private final OptionsPresenter optionsPresenter;
 
     public TopTabsLayout(Context context, List<TopTabController> tabs, TopTabsAdapter adapter) {
         super(context);
-        topBar = new TopBar(context, this);
+        this.tabs = tabs;
+        topBar = new TopBar(context, this, null);
```

Comment:
Avoid passing null, better to overload and pass null internally.

Reply:
We should open an issue to implement collapse in topTabs screen

Reply:
Yes